frontend/package-lock.json

```diff
@@ -59498,4 +59498,4 @@
       "dev": true
     }
   }
-}
+}
```

Comment:
This change should not have been committed. It removes a trailing newline from a file which otherwise wasn't edited.

Reply:
I see the red warning marker in other files here too. Set up your IDE settings to add the trailing newline on save and you'll be sorted.